queryscript/src/compile/schema.rs

```diff
@@ -819,6 +819,7 @@ where
     ContextRef(Ident),
     Connection(Arc<ConnectionString>),
     Materialize(MaterializeExpr<TypeRef>),
+    UncompiledFn(ast::FnDef),
```

Comment:
not something we should do in this change but another good piece of evidence that we should have a separate runtime expr tree i think